synapse/lib/ast.py

```diff
@@ -997,8 +997,10 @@ async def genx():
                         raise s_stormctrl.StormExit()
 
                 if runtsafe or not empty:
-                    async for item in scmd.execStormCmd(runt, genr):
-                        yield item
+                    async with contextlib.aclosing(scmd.execStormCmd(runt, genr)) as agen:
```

Comment:
Seeing the delta in c165c58fa33c5d35af3e7446d16646a847cce2c1 makes me wonder if we need to keep using s_common.aclosing at all? Should we replace that with contextlib.aclosing now that is available to us?

Reply:
Yeah we can swap it out now that we are on 3.11+ only